pages/dkp/kommander/1.4/clusters/attach-cluster/generate-kubeconfig/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Generate a kubeconfig File
+title: Generate a kubeconfig File
+menuWeight: 1
+excerpt: How to create a service account and generate a kubeconfig file for attaching an existing cluster
+---
+
+You should create a separate service account when attaching existing Amazon EKS, Azure AKS, or Google GKE Kubernetes clusters. This service account is needed because the kubeconfig files generated from those clusters are not usable by Kommander out-of-the-box. They call CLI commands, such as `aws` or `gcloud`, and use locally-obtained, authentication tokens. Having a separate service account also allows you to keep access to the cluster specific and isolated to Kommander.
```

Comment:
```suggestion
You should create a separate service account when attaching existing Amazon EKS, Azure AKS, or Google GKE Kubernetes clusters. This service account is needed because the kubeconfig files generated from those clusters are not usable by Kommander out-of-the-box. They call CLI commands, such as `aws` or `gcloud`, and use locally-obtained authentication tokens. Having a separate service account also allows you to keep access to the cluster specific and isolated to Kommander.
```

Reply:
Changed as noted under D2IQ-74942.